lib/main.coffee

```diff
@@ -296,36 +288,49 @@ module.exports = Hydrogen =
             @moveDown endRow
 
     removeStatusBarElement: ->
-        if @statusBarElement?
-            while @statusBarElement.hasChildNodes()
-                @statusBarElement.removeChild(@statusBarElement.lastChild)
-
-    setStatusBarElement: (element) ->
-        if @statusBarElement?
-            @removeStatusBarElement()
-            @statusBarElement.appendChild(element)
-        else
-            console.error "No status bar element. Can't set it."
+        unless @statusBarElement?
+            console.error 'removeStatusBarElement: there is no status bar'
+            return
+
+        while @statusBarElement.hasChildNodes()
+            @statusBarElement.removeChild @statusBarElement.lastChild
+
+    setStatusBarElement: ->
+        unless @statusBarElement?
+            console.error 'setStatusBarElement: there is no status bar'
```

Comment:
Oh! I see the problem. This is actually a console error. It's happening because I dropped [this check](https://github.com/nteract/hydrogen/blob/20be0e3a22d296a2f282a17fa9d1ad777d222126/lib/main.coffee#L95). I'll fix it.